linera-service/src/config.rs

```diff
@@ -125,10 +125,11 @@ impl UserChain {
         }
     }
 
-    /// Creates an entry for a chain that we don't own.
-    pub fn make_other(description: ChainDescription, timestamp: Timestamp) -> Self {
+    /// Creates an entry for a chain that we don't own. The timestamp must be the genesis
+    /// timestamp or earlier.
```

Comment:
Should we just set the timestamp to 0 (i.e. January 1970), since the common case is that we don't know it at this point, and it will be updated with the first block anyway?

Reply:
So right now, we actually know the genesis timestamp at the caller site, but I suppose we could change if needed